code/AssetLib/Blender/BlenderScene.h

```diff
@@ -729,7 +728,7 @@ struct Object : ElemBase {
     ListBase modifiers;
 
     Object() :
-            ElemBase(), type(Type_EMPTY), parent(nullptr), track(), proxy(), proxy_from(), data() {
+            type(Type_EMPTY), parent(nullptr) {
```

Comment:
base class default c’tors are always called; `std::shared_ptr` is always default-initialized